specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/examples/SuccessfulEntitiesRequest.json

```diff
@@ -0,0 +1,91 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "Endpoint": "{Endpoint}",
+    "input": {
+      "documents": [
+        {
+          "language": "en",
+          "id": "1",
+          "text": "I had a wonderful trip to Seattle last week."
+        },
+        {
+          "language": "en",
+          "id": "2",
+          "text": "I work at Microsoft."
+        },
+        {
+          "language": "en",
+          "id": "3",
+          "text": "I visited Space Needle 2 times."
+        }
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "documents": [
+          {
+            "id": "1",
+            "entities": [
+              {
+                "text": "Seattle",
+                "type": "Location",
+                "offset": 26,
+                "length": 7,
+                "score": 0.8062429428100586
```

Comment:
This will be updated to confidenceScore for v3.0 (non-preview). Additionally, the score will be rounded to 2 decimal places.